File: 2.0/csharp/contacts/findDoNotContacts.cs
using CallfireApiClient;
using CallfireApiClient.Api.Common.Model;
using CallfireApiClient.Api.Contacts.Model;
using CallfireApiClient.Api.Contacts.Model.Request;
using System.Collections.Generic;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new FindDncNumbersRequest()
        {
            Prefix = "1424",
            Numbers = new List<string> { "12135551126", "12136666123" },
            CampaignId = 11646003,
            Source = "my source",
            Call = true,
            Text = true,
            Offset = 0,
            Limit = 10,
            Fields = "items(number,call)"
        };
        Page<DoNotContact> dncs = client.DncApi.Find(request);
    }
}

File: 2.0/java/contacts/findDoNotContacts.java
import java.util.Arrays;

import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.common.model.Page;
import com.callfire.api.client.api.contacts.model.DoNotContact;
import com.callfire.api.client.api.contacts.model.request.FindDncNumbersRequest;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        FindDncNumbersRequest request = FindDncNumbersRequest.create()
                .prefix("1424")
                .numbers(Arrays.asList("12135551126", "12136666123"))
                .campaignId(11646003L)
                .source("my source")
                .call(true)
                .text(true)
                .offset(0L)
                .limit(10L)
                .fields("items(number,call)")
                .build();
        Page<DoNotContact> dncContacts = client.dncApi().find(request);
    }
}

